<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimeraEntrega</class>
 <widget class="QWidget" name="PrimeraEntrega">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="4" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="gbInfo">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Datos extra√≠dos</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0" columnstretch="0,0,1,1">
        <item row="6" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="Visor" name="visorFotoPerfil">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Foto de perfil</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="2" colspan="2">
         <widget class="QLineEdit" name="leVencimiento"/>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="QLineEdit" name="leOtorgamiento"/>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="QLineEdit" name="leNacimiento"/>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QLineEdit" name="leNombre"/>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLineEdit" name="leApellido"/>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QLineEdit" name="leDni"/>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>DNI</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Apellido</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Nombre</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Nacimiento</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Otorgamiento</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Vencimiento</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pbElegirFoto">
     <property name="text">
      <string>Elegir foto</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="pbProcesar">
     <property name="text">
      <string>Procesar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="2" colspan="4">
    <widget class="Visor" name="visor">
     <property name="text">
      <string>Foto para procesar</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Visor</class>
   <extends>QLabel</extends>
   <header>visor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
